README.md

```diff
@@ -1,92 +1,119 @@
-[![Apache License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
-# Asana 
-
-This package models Asana data from [Fivetran's connector](https://fivetran.com/docs/applications/asana). It uses data in the format described by [this ERD](https://fivetran.com/docs/applications/asana#schemainformation).
-
-This package enriches your Fivetran data by doing the following:
-* Adds descriptions to tables and columns that are synced using Fivetran
-* Adds freshness tests to source data
-* Adds column-level testing where applicable. For example, all primary keys are tested for uniqueness and non-null values.
-* Models staging tables, which will be used in our transform package
-
-## Models
-
-This package contains staging models, designed to work simultaneously with our [Asana transform package](https://github.com/fivetran/dbt_asana).  The staging models:
-* Remove any rows that are soft-deleted
-* Name columns consistently across all packages:
+<p align="center">
+    <a alt="License"
+        href="https://github.com/fivetran/dbt_asana_source/blob/main/LICENSE">
+        <img src="https://img.shields.io/badge/License-Apache%202.0-blue.svg" /></a>
+    <a alt="dbt-core">
+        <img src="https://img.shields.io/badge/dbt_Core™_version->=1.0.0_<2.0.0-orange.svg" /></a>
+    <a alt="Maintained?">
+        <img src="https://img.shields.io/badge/Maintained%3F-yes-green.svg" /></a>
+    <a alt="PRs">
+        <img src="https://img.shields.io/badge/Contributions-welcome-blueviolet" /></a>
+</p>
+
+# Asana Source dbt Package ([Docs](https://fivetran.github.io/dbt_asana_source/))
+# 📣 What does this dbt package do?
+- Materializes [Asana staging tables](https://fivetran.github.io/dbt_asana_source/#!/overview/asana_source/models/?g_v=1&g_e=seeds), which leverage data in the format described by [this ERD](https://fivetran.com/docs/applications/asana#schemainformation). These staging tables clean, test, and prepare your Asana data from [Fivetran's connector](https://fivetran.com/docs/applications/asana) for analysis by doing the following:
+  - Names columns for consistency across all packages and for easier analysis
     * Boolean fields are prefixed with `is_` or `has_`
     * Timestamps are appended with `_at`
     * ID primary keys are prefixed with the name of the table.  For example, the user table's ID column is renamed user_id.
     * Foreign keys include the table that they refer to. For example, the project table's owner ID column is renamed owner_user_id.
```

Comment:
We started to remove these bullets since they added more details than we really need to include. We can remove them from the README.

Reply:
removed!